udata/core/dataset/models.py

```diff
@@ -747,6 +750,23 @@ def from_community(self):
         return True
 
 
+class ResourceSchemas(object):
+    @staticmethod
+    def get():
+        endpoint = current_app.config.get('SCHEMA_CATALOG_URL')
+        if endpoint is None:
+            return []
+
+        r = requests.get(endpoint)
+        r.raise_for_status()
```

Comment:
I think we need a smarter caching strategy and exception handling.

Something like that https://github.com/etalab/udata-gouvfr/pull/486/files#diff-1c9abf3241dab91b0e60c554f8bcd67aR72.

This would lead to removing the cache on the API and handle it here.